TrafficController unit test failure

**Describe the bug**
<!--
A clear and concise description of what the bug is.
If you believe this bug is a security issue, please don't use this template and follow our [security guidelines](/SECURITY.md)
-->
Hi,

I hit this failure when running unit test. It seems a flaky failure as I can not reproduce it every time.

```
W0930 02:31:50.152991    8912 env.go:126] Failed to get Pod Namespace from environment. Using "kube-system" as the Antrea Service Namespace
--- FAIL: TestSharedTargetPort (1.10s)
    controller_test.go:282: 
                Error Trace:    /usr/src/antrea.io/antrea/pkg/agent/controller/trafficcontrol/controller_test.go:282
                                                        /usr/src/antrea.io/antrea/pkg/agent/controller/trafficcontrol/controller_test.go:666
                Error:          Received unexpected error:
                                timed out waiting for the condition
                Test:           TestSharedTargetPort
    controller.go:269: missing call(s) to *testing.MockClient.UninstallTrafficControlMarkFlows(is equal to test-tc1 (string)) /usr/src/antrea.io/antrea/pkg/agent/controller/trafficcontrol/controller_test.go:658
    controller.go:269: missing call(s) to *testing.MockClient.UninstallTrafficControlMarkFlows(is equal to test-tc2 (string)) /usr/src/antrea.io/antrea/pkg/agent/controller/trafficcontrol/controller_test.go:659
    controller.go:269: missing call(s) to *testing.MockOVSBridgeClient.DeletePort(is anything) /usr/src/antrea.io/antrea/pkg/agent/controller/trafficcontrol/controller_test.go:660
    controller.go:269: aborting test due to missing call(s)
I0930 02:31:59.752502    8912 channel.go:83] "Starting SubscribableChannel" name="PodUpdate"
I0930 02:31:59.762943    8912 controller.go:911] "Interfaces of Pod not found" Pod="ns1/pod1"
I0930 02:31:59.774413    8912 channel.go:87] "Stopping SubscribableChannel" name="PodUpdate"
FAIL
coverage: 2.1% of statements in antrea.io/antrea/cmd/..., antrea.io/antrea/pkg/..., antrea.io/antrea/multicluster/cmd/..., antrea.io/antrea/multicluster/controllers/...
FAIL    antrea.io/antrea/pkg/agent/controller/trafficcontrol    12.451s
```

**To Reproduce**
<!-- Steps to reproduce the behavior. -->

**Expected**
<!-- A clear and concise description of what you expected to happen. -->

**Actual behavior**
<!-- A clear and concise description of what's the actual behavior. If applicable, add screenshots, log messages, etc. to help explain the problem. -->

**Versions:**
<!--
Please provide the following information:
 - Antrea version (Docker image tag).
 - Kubernetes version (use `kubectl version`). If your Kubernetes components have [different versions](https://kubernetes.io/docs/setup/release/version-skew-policy/), please provide the version for all of them.
 - Container runtime: which runtime are you using (e.g. containerd, cri-o, docker) and which version are you using?
 - Linux kernel version on the Kubernetes Nodes (`uname -r`).
 - If you chose to compile the Open vSwitch kernel module manually instead of using the kernel module built into the Linux kernel, which version of the OVS kernel module are you using? Include the output of `modinfo openvswitch` for the Kubernetes Nodes.
-->

Antrea: main branch

**Additional context**
<!-- Add any other context about the problem here, such as Antrea logs, kubelet logs, etc. -->

<!-- (Please consider pasting long output into a [GitHub gist](https://gist.github.com) or any other pastebin.) -->

--- a/pkg/agent/controller/trafficcontrol/controller_test.go
+++ b/pkg/agent/controller/trafficcontrol/controller_test.go
@@ -279,7 +279,7 @@ func generateTrafficControlState(direction v1alpha2.Direction,
 }
 
 func waitEvents(t *testing.T, expectedEvents int, c *fakeController) {
-	require.NoError(t, wait.PollImmediate(10*time.Millisecond, time.Second, func() (done bool, err error) {
+	require.NoError(t, wait.PollImmediate(10*time.Millisecond, 5*time.Second, func() (done bool, err error) {
 		return c.queue.Len() == expectedEvents, nil
 	}))
 }

